src/conn/conn_api.c

```diff
@@ -1880,12 +1888,16 @@ __conn_single(WT_SESSION_IMPL *session, const char *cfg[])
      * above, locked files cannot be copied on Windows). If the WiredTiger
      * file exists in the directory, create the lock file, covering the case
      * of a hot backup.
+     *
+     * In addition, in disagg mode, we should create the lock file regardless
+     * of whether the  WiredTiger file exists or not because WiredTiger may
+     * not be there.
```

Comment:
I don't quite understand this comment. What does it mean by "WiredTiger may not be there"?

Reply:
I reworded that to "WiredTiger **file**"

Reply:
Thanks! That makes more sense now.